app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3 and command.split()[1] != command.split()[2]:
+        mv, file_in, file_out = command.split()
```

Comment:
DRY - don't repeat yourself 

`command.split()` - you only have to do it once, now you do it 4 times